dev/com.ibm.ws.jsf.2.2_fat/test-applications/ViewScopeLeak.war/resources/index.xhtml

```diff
@@ -26,9 +23,7 @@
             <h:form id="form1">
                 <h:outputText id="out1" value="#{ViewScopedCDIBean.value}"/>
                 <p/>
-                <h:commandButton id="ajaxButton" value="Count CDI" action="#{ViewScopedCDIBean.increment}">
-                    <f:ajax render="out1"/>
-                </h:commandButton>
+                <h:commandButton id="ajaxButton" value="Count CDI" action="#{ViewScopedCDIBean.increment}"/>
```

Comment:
Removing the Ajax that isn't necessary for this test so we can continue to test with HTMLUnit